browser/src/Editor/NeovimEditor.tsx

```diff
@@ -256,6 +262,12 @@ export class NeovimEditor implements IEditor {
                 }
             }
 
+            if (key === "<C-v>" && this._screen.mode === "insert") {
```

Comment:
Yes, good point, thanks! I just added a check for it around the keys that we are strong-arming into clipboard behavior when this setting is enabled (so disabling it should actually work now). This will be an interesting case for us to untangle with the input binding work, as:
- These should be applied as defaults (prior to loading the keybindings), if the `editor.clipboard.enabled` setting is true.
- There will be conditional applications based on the platform (Windows vs OSX vs Linux) - not too different from how we handle default configuration values today, though.
- They will need to be filtered / enabled based on the mode, and either execute an Oni-side command (`editor.clipboard.paste`), or a new set of keystrokes to Neovim (`y`).